tests/data/bundles/test_yahoo.py

```diff
@@ -1,11 +1,13 @@
+from __future__ import division
```

Comment:
@llllllllll thinks he added this to the PR intentionally to make this module compatible with both python 2 and 3.